content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,221 +11,196 @@ weight: 10
 
 <!-- overview -->
 
-On-disk files in a container are ephemeral, which presents some problems for
-non-trivial applications when running in containers. One problem
-is the loss of files when a container crashes. The kubelet restarts the container
-but with a clean state. A second problem occurs when sharing files
-between containers running together in a `Pod`.
-The Kubernetes {{< glossary_tooltip text="volume" term_id="volume" >}} abstraction
-solves both of these problems.
-Familiarity with [Pods](/docs/concepts/workloads/pods/) is suggested.
+File storage is useful when building stateful applications. Use _volumes_ to declaratively manage storage for containers. The Kubernetes {{< glossary_tooltip text="volume" term_id="volume" >}} abstraction allows files to persist even beyond the lifetime of a pod. Volumes also allow different containers within a pod to share storage and to access configuration.
```

Comment:
File storage is sometimes also useful when building stateless applications. For example, an `emptyDir` volume can be used for a cache, or a Pod might mount service account data via a volume.
I recommend rewording to avoid implying that only stateful apps would use volumes.

Reply:
good point